.gitlab-ci.yml

```diff
@@ -7,7 +7,7 @@ stages:
   - trigger
 
 variables:
-  GOLANG_VERSION: "1.17.6"
+  GOLANG_VERSION: "1.17.13"
```

Comment:
could you please verify that we test with the same version we build?

Reply:
not relevant